src/cli.py

```diff
@@ -24,3 +25,11 @@ async def cli(args, script_root: Path):
             print("All exercises finished ! 🎉")
         else:
             print(exercise_path)
+
+    if args.solution:
+        displayer = SolutionPatcher(args.solution)
+        opt_solution = displayer.get_solution()
```

Comment:
This is unclear to me what `opt_solution` stands for. `solution_display` maybe? Or `formatted_solution`? opt usually stands for `option`